%% Template for a CV
%% Author: Rob J Hyndman

\documentclass[10pt,a4paper,]{article}
\usepackage[scaled=0.86]{DejaVuSansMono}
\usepackage[sfdefault,lf,t]{carlito}
\usepackage{fancyhdr}
\pagestyle{fancy}

% Change color to blue
\usepackage{color,xcolor}
\definecolor{headcolor}{HTML}{990000}

\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi

\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}

% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage[]{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\PassOptionsToPackage{hyphens}{url} % url is loaded by hyperref
\usepackage[unicode=true,hidelinks]{hyperref}
\urlstyle{same}  % don't use monospace font for urls
\usepackage{geometry}
\geometry{left=1.75cm,right=1.75cm,top=2.2cm,bottom=2cm}

\usepackage{longtable,booktabs}
% Fix footnotes in tables (requires footnote package)
\IfFileExists{footnote.sty}{\usepackage{footnote}\makesavenoteenv{long table}}{}
\IfFileExists{parskip.sty}{%
\usepackage{parskip}
}{% else
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{0}

% set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother



\date{April 2022}


\usepackage{paralist,ragged2e,datetime}
\usepackage[hyphens]{url}
\usepackage{fancyhdr,enumitem,pifont}
\usepackage[compact,small,sf,bf]{titlesec}

\RaggedRight
\sloppy

% Use fontawesome. Note: you'll need TeXLive 2015. Update.


\def\maketitle{
\vspace*{-1.4cm}
\thispagestyle{empty}
\centerline{\huge \bf Chase Clark}

%job title

\vspace{1 mm}
\centerline{
%address
\faicon{map-marker} University of Wisconsin-Madison, School of Pharmacy,
VCRGE/BIOTECH/GENOMICS \hspace{1 mm} 
}
\vspace{1 mm}

%github
\centerline{
\faicon{github} \href{http://github.com/chasemc}{\color{black} chasemc} \hspace{1 mm} %
%twitter
\faicon{twitter} \href{http://twitter.com/ChasingMicrobes}{\color{black} ChasingMicrobes} \hspace{1 mm} %
%phone
%
%email
\faicon{envelope} \href{mailto:}{\color{black} \href{mailto:chasingmicrobes@gmail.com}{\nolinkurl{chasingmicrobes@gmail.com}}} \hspace{1 mm} %
%linkedin
\faicon{linkedin} \href{https://www.linkedin.com/in/hirechase}{hirechase} \hspace{1 mm} %
%scholar
%
%orcid
%
%website
\faicon{home} \href{http://chasemc.github.io}{chasemc.github.io}\hspace{1 mm} %
}

\vspace{2 mm}
\hrule
}
% Header and footer
\pagestyle{fancy}
\makeatletter
\lhead{\sf\textcolor[gray]{0.4}{Resume: \@name}}
\rhead{\sf\textcolor[gray]{0.4}{\thepage}}
\cfoot{}
\def\headrule{{\color[gray]{0.4}\hrule\@height\headrulewidth\@width\headwidth \vskip-\headrulewidth}}
\makeatother

% Header box
\usepackage{tabularx}

\makeatletter
\def\name#1{\def\@name{#1}}
\def\info#1{\def\@info{#1}}
\makeatother
% \newcommand{\shadebox}[3][.9]{\fcolorbox[gray]{0}{#1}{\parbox{#2}{#3}}}
% 
% \usepackage{calc}
% \newlength{\headerboxwidth}
% \setlength{\headerboxwidth}{\textwidth}
% %\addtolength{\headerboxwidth}{0.2cm}
% \makeatletter
% \def\maketitle{
% \thispagestyle{plain}
% \vspace*{-1.4cm}
% \shadebox[0.9]{\headerboxwidth}{\sf\color{headcolor}\hfil
% \hbox to 0.98\textwidth{\begin{tabular}{l}
% \\[-0.3cm]
% \LARGE\textbf{\@name}
% \\[0.7cm]
% \normalsize\textbf{Curriculum Vitae}\\
% \normalsize April 2022
% \end{tabular}
% \hfill\hbox{\fontsize{9}{12}\sf
% \begin{tabular}{@{}rl@{}}
% \@info
% \end{tabular}}}\hfil
% }
% \vspace*{0.2cm}}
% \makeatother

% Section headings
\titlelabel{}
\titlespacing{\section}{0pt}{1.5ex}{0.5ex}
\titleformat*{\section}{\large\sf\bfseries}
\titleformat*{\subsection}{\sf\bfseries}
\titlespacing{\subsection}{0pt}{1ex}{0.5ex}

% Miscellaneous dimensions
\setlength{\parskip}{0ex}
\setlength{\parindent}{0em}
\setlength{\headheight}{15pt}
\setlength{\tabcolsep}{0.15cm}
\clubpenalty = 10000
\widowpenalty = 10000
\setlist{itemsep=2pt}
\setdescription{labelwidth=1.2cm,leftmargin=1.5cm,labelindent=1.5cm,font=\rm}

% % Make nicer bullets
% \renewcommand{\labelitemi}{\ding{228}}

\usepackage{booktabs,fontawesome}
%\usepackage[t1,scale=0.86]{sourcecodepro}

\name{Chase Clark}
\def\imagetop#1{\vtop{\null\hbox{#1}}}
\info{%
\raisebox{-0.05cm}{\imagetop{\faicon{map-marker}}} &  \imagetop{\begin{tabular}{@{}l@{}}University
of Wisconsin-Madison, School of Pharmacy,
VCRGE/BIOTECH/GENOMICS\end{tabular}}\\ %
\faicon{home} & \href{http://chasemc.github.io}{chasemc.github.io}\\% %
%
\faicon{envelope} & \href{mailto:chasingmicrobes@gmail.com}{\nolinkurl{chasingmicrobes@gmail.com}}\\%
\faicon{twitter} & \href{https://twitter.com/ChasingMicrobes}{@ChasingMicrobes}\\%
\faicon{github} & \href{https://github.com/chasemc}{chasemc}\\%
\faicon{linkedin} & \href{https://www.linkedin.com/in/hirechase}{hirechase}\\%
%
%
}


%\usepackage{inconsolata}


\setlength\LTleft{0pt}
\setlength\LTright{0pt}

% Pandoc CSL macros
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{3em}
\newenvironment{CSLReferences}[3] % #1 hanging-ident, #2 entry spacing
 {% don't indent paragraphs
  \setlength{\parindent}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1 \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces\fi
  % set entry spacing
  \ifnum #2 > 0
  \setlength{\parskip}{#2\baselineskip}
  \fi
 }%
 {}
\usepackage{calc}
\newcommand{\CSLBlock}[1]{#1\hfill\break}
\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\csllabelwidth}{\hfill #1~}}
\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \cslhangindent - \csllabelwidth}{#1}\vspace{0.8ex}}
\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}


\def\endfirstpage{\newpage}

\begin{document}
\maketitle


\hypertarget{employment}{%
\section{Employment}\label{employment}}

\begin{longtable}{@{\extracolsep{\fill}}ll}
2020 & \parbox[t]{0.85\textwidth}{%
\textbf{PhD Pharmacognosy}\hfill{University of Illinois at Chicago}\newline
  \footnotesize Chicago, IL\par%
  \normalsize \empty%
\vspace{1.5\parsep}}\\
2012 & \parbox[t]{0.85\textwidth}{%
\textbf{BS Biochemistry}\hfill{Berry College}\newline
  \footnotesize Mount Berry, GA\par%
  \normalsize \empty%
\vspace{1.5\parsep}}\\
2010 & \parbox[t]{0.85\textwidth}{%
\textbf{Principia Consortium Study Abroad}\hfill{University of Glasgow}\newline
  \footnotesize Glasgow, Glasgow\par%
  \normalsize \empty%
\vspace{1.5\parsep}}\\
\end{longtable}

\hypertarget{experience}{%
\section{Experience}\label{experience}}

\begin{longtable}{@{\extracolsep{\fill}}ll}
Jun 2021 - 
present & \parbox[t]{0.85\textwidth}{%
\textbf{Computation and Informatics in Biology and Medicine (CIBM) Postdoctoral Fellow}\hfill{}\newline
  \footnotesize Remote, Chicago, IL\par%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item Training grant funded through the National Library of Medicine%
\item Developing genomic and metagenomic bioinformatics software%
\item Currently developing a repository scale analysis platform (synthesizing tens of billions of data points)%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
Sep 2020 - 
present & \parbox[t]{0.85\textwidth}{%
\textbf{Postdoctoral Research Associate}\hfill{University of Wisconsin-Madison}\newline
  \footnotesize Remote, Chicago, IL\par%
  \normalsize \empty%
\vspace{1.5\parsep}}\\
May 2019 - 
Aug 2020 & \parbox[t]{0.85\textwidth}{%
\textbf{F31 Fellow}\hfill{U.S. National Institutes of Health (NIH)}\newline
  \footnotesize Chicago, IL, US\par%
  \normalsize \empty%
\vspace{1.5\parsep}}\\
Aug 2015 - 
Aug 2020 & \parbox[t]{0.85\textwidth}{%
\textbf{Ph.D. Student}\hfill{University of Illinois at Chicago}\newline
  \footnotesize Chicago, IL, US\par%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item Natural product drug dicovery, specifically bacterial-derived antibiotics%
\item Conceived, learned to code, and built a bioinformatics program (R Shiny app) for rapid bacterial metabolomics analyses%
\item Isolated and cultured 1000’s of bacterial isolates and performed natural product isolation and structure elucidation%
\item First or co-first author on four studies/manuscripts%
\item Mentored undergradaute and graduate students leading to publishable studies%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
Mar 2013 - 
Aug 2015 & \parbox[t]{0.85\textwidth}{%
\textbf{Research and Development Technician}\hfill{Deerland Probiotics \& Enzymes}\newline
  \footnotesize Kennesaw, GA, US\par%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item Assisted in the design and results analysis of three clinical studies%
\item Presented monthly projects to CEO, VP Sales and VP Science \& Technology%
\item Head of method development and identification of raw materials to meet CFR 21, FDA requirements.%
\item Developed identifications for over 140 materials, from enzymes and botanicals to minerals and additives. %
\item Overhauled the Method Development Department establishing inter-departmental responsibilities, improving and creating new SOPs, streamlining identification requests by establishing a formal request  process and interactive database%
\item Strengthened and created new relationships with third-party vendors, labs, and universities. %
\item Controlled budget of method development for purchase of standards, consumables, equipment and third-party testing.%
\item Mentored interns from Kennesaw Mountain High School magnet program%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
Aug 2008 - 
May 2012 & \parbox[t]{0.85\textwidth}{%
\textbf{Bonner Scholar}\hfill{Bonner Foundation}\newline
  \footnotesize Rome, GA, US\par%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item Four-year community service scholarship requiring weekly and summer commitments.%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
\end{longtable}
\newpage

\hypertarget{data-science-skills}{%
\section{Data Science Skills}\label{data-science-skills}}

\textbf{Bioinformatics}: metabomics () \(\bullet\) genomics ()
\(\bullet\) metagenomics ()

\textbf{Cloud Computing}: remote server computing (ssh, etc) \(\bullet\)
high-performance and high-throughput computing (Open science grid,
HTCondor)

\textbf{Communication}: presentations (public speaking, technical
writing,peer-reviewed publications, SOPs) \(\bullet\) writing
(customer-facing documents, peer-reviewed publications, SOPs)
\(\bullet\) full-stack design (Shiny, Flask,Django) \(\bullet\) reports
(Rmarkdown, Jupyter) \(\bullet\) visualization (Base R, ggplot2, plotly,
leaflet, cytoscape, networkx, gephi, etc)

\textbf{Programming}: general (R, Python) \(\bullet\) database (Neo4j,
SQL) \(\bullet\) automation (Nextflow, Make) \(\bullet\) containers
(Docker) \(\bullet\) pipeline (Nextflow, Make)

\textbf{Software Development}: source control (Git) \(\bullet\) CI/CD
(Github, Travis, Azure, DevOps, etc.) \(\bullet\) automated testing
(testthat, pytest, etc)

\textbf{Statistics}: machine learning () \(\bullet\) data analysis ()
\(\bullet\) cluster analysis () \(\bullet\) factor analysis ()
\(\bullet\) principal components analysis () \(\bullet\)
cross-validation () \(\bullet\) experimental design ()

\hypertarget{select-individual-software-projects}{%
\section{Select Individual Software
Projects}\label{select-individual-software-projects}}

\begin{longtable}{@{\extracolsep{\fill}}ll}
2021 & \parbox[t]{0.85\textwidth}{%
\textbf{socialgene}\hfill{Python, Django, Nextflow, High-throughput computing}\newline
  \footnotesize Repository-scale genome pattern search platform\par%
  \normalsize \empty%
\vspace{1.5\parsep}}\\
2019 & \parbox[t]{0.85\textwidth}{%
\textbf{electricShine}\hfill{R, Javascript, Node/Electron}\newline
  \footnotesize Shiny packaging with Electron\par%
  \normalsize \empty%
\vspace{1.5\parsep}}\\
2019 & \parbox[t]{0.85\textwidth}{%
\textbf{mzEasy}\hfill{R, Shiny}\newline
  \footnotesize Shiny app for converting and visualizing mass spectrometry data\par%
  \normalsize \empty%
\vspace{1.5\parsep}}\\
2019 & \parbox[t]{0.85\textwidth}{%
\textbf{mzPlotter}\hfill{R, Rmarkdown}\newline
  \footnotesize Automated summary of of LC-MS/MS data with interactive Rmarkdown reports\par%
  \normalsize \empty%
\vspace{1.5\parsep}}\\
2019 & \parbox[t]{0.85\textwidth}{%
\textbf{mzFromImage}\hfill{R, Shiny}\newline
  \footnotesize Predict a mass spectrum’s values from a static image file\par%
  \normalsize \empty%
\vspace{1.5\parsep}}\\
2019 & \parbox[t]{0.85\textwidth}{%
\textbf{mgfparse}\hfill{R}\newline
  \footnotesize Low-dependency R package for quickly and efficiently parsing mgf files into R\par%
  \normalsize \empty%
\vspace{1.5\parsep}}\\
2018 & \parbox[t]{0.85\textwidth}{%
\textbf{IDBac}\hfill{R, Shiny, Electron}\newline
  \footnotesize MALDI protein and small molecule bioinformatics platform\par%
  \normalsize \empty%
\vspace{1.5\parsep}}\\
\end{longtable}

\hypertarget{select-team-software-projects}{%
\section{Select Team Software
Projects}\label{select-team-software-projects}}

\begin{longtable}{@{\extracolsep{\fill}}ll}
2021 & \parbox[t]{0.85\textwidth}{%
\textbf{Autometa}\hfill{Python, Nextflow}\newline
  \footnotesize Automated binning pipeline for single metagenomes\par%
  \normalsize \empty%
\vspace{1.5\parsep}}\\
2021 & \parbox[t]{0.85\textwidth}{%
\textbf{metaBenchmarks}\hfill{Nextflow}\newline
  \footnotesize Benchmark metagenomic profiling/binning software\par%
  \normalsize \empty%
\vspace{1.5\parsep}}\\
\end{longtable}

\hypertarget{talks}{%
\section{Talks}\label{talks}}

\begin{longtable}{@{\extracolsep{\fill}}ll}
2022-03-09 & \parbox[t]{0.85\textwidth}{%
\textbf{Gordon Research Conference on Marine Natural Products}\hfill{}\newline
  \footnotesize \empty%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item Towards near-instant, repository-scale searching for homologous BGCs with Socialgene (Selected alternate speaker, didn't present)%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
2020-06-01 & \parbox[t]{0.85\textwidth}{%
\textbf{St. Jude National Graduate Student Symposium (cancelled due to Covid19)}\hfill{}\newline
  \footnotesize \empty%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item IDBac: Bioinformatics Software for Microbial Drug Discovery Prioritization and Culturomics Characterization%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
2020-05-08 & \parbox[t]{0.85\textwidth}{%
\textbf{May Institute 2020: Future developers meeting}\hfill{}\newline
  \footnotesize \empty%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item Protein MS isn’t the only MS… Programming in R for Metabolomics Mass Spectrometry%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
2020-02-25 & \parbox[t]{0.85\textwidth}{%
\textbf{Gordon Research Conference on Marine Natural Products}\hfill{}\newline
  \footnotesize \empty%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item IDBac: Bioinformatics Software for Microbial Drug Discovery Prioritization and Culturomics Characterization%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
2020-02-23 & \parbox[t]{0.85\textwidth}{%
\textbf{Gordon Research Seminar (GRS) on Marine Natural Products}\hfill{}\newline
  \footnotesize \empty%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item IDBac: Bioinformatics Software for Microbial Drug Discovery Prioritization and Culturomics Characterization%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
2019-08-23 & \parbox[t]{0.85\textwidth}{%
\textbf{R/Pharma, Harvard University}\hfill{}\newline
  \footnotesize \empty%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item Your Missing Step in Reproducible R Programming: Continuous Deployment%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
2019-04-27 & \parbox[t]{0.85\textwidth}{%
\textbf{satRdays Chicago}\hfill{}\newline
  \footnotesize \empty%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item Bioinformatics in R%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
2019-01-23 & \parbox[t]{0.85\textwidth}{%
\textbf{Chicago R User Group}\hfill{}\newline
  \footnotesize \empty%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item Single Function Lightning Talks: “lengths(), not length()”%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
2018-08-15 & \parbox[t]{0.85\textwidth}{%
\textbf{R/Pharma, Harvard University}\hfill{}\newline
  \footnotesize \empty%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item IDBac: A New Paradigm in Developing Microbial Libraries for Drug Discovery%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
2018-01-27 & \parbox[t]{0.85\textwidth}{%
\textbf{Chicago R User Group}\hfill{}\newline
  \footnotesize \empty%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item IDBac: A Shiny App to Analyze Bacterial Fingerprints and Aid in the Discovery of Potential New Antibiotics%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
2017-03-09 & \parbox[t]{0.85\textwidth}{%
\textbf{Center for Biomolecular Sciences, UIC}\hfill{}\newline
  \footnotesize \empty%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item IDBac: A proteomic \& chemometric pipeline for rapid bacterial characterization%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
2016-11-03 & \parbox[t]{0.85\textwidth}{%
\textbf{UIC Specialized Metabolite Community}\hfill{}\newline
  \footnotesize \empty%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item Rational design of bacterial strain libraries for drug discovery%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
2016-07-22 & \parbox[t]{0.85\textwidth}{%
\textbf{Chicago Mass Spec Day}\hfill{}\newline
  \footnotesize \empty%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item Use of MALDI-MS to Create ‘Smart’ Libraries for Drug-Lead Discovery%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
2015-04-15 & \parbox[t]{0.85\textwidth}{%
\textbf{SSAOAC Annual Meeting}\hfill{}\newline
  \footnotesize \empty%
  \normalsize \vspace{0.1cm}\begin{minipage}{0.7\textwidth}%
\begin{itemize}[leftmargin=*]%
\item HPLC-DAD Method for Trace Detection of Benzoic and Sorbic Acids in High Protein Matrices%
\end{itemize}%
\end{minipage}%
\vspace{1.5\parsep}}\\
\end{longtable}

\hypertarget{awards}{%
\section{Awards}\label{awards}}

\begin{longtable}{@{\extracolsep{\fill}}ll}
2021-2023 & \parbox[t]{0.85\textwidth}{%
\textbf{Computation and Informatics in Biology and Medicine (CIBM) training}\\[-0.1cm]{\footnotesize \$100,000.00}}\\[0.4cm]
2019-2020 & \parbox[t]{0.85\textwidth}{%
\textbf{NIH Predoctoral Individual National Research Service Award (F31)}\\[-0.1cm]{\footnotesize \$90,000.00}}\\[0.4cm]
2017 & \parbox[t]{0.85\textwidth}{%
\textbf{American Society of Pharmacognosy Student Travel Award}\\[-0.1cm]{\footnotesize \$600}}\\[0.4cm]
2017 & \parbox[t]{0.85\textwidth}{%
\textbf{University of Illinois at Chicago Provost Deiss Award}\\[-0.1cm]{\footnotesize \$2,500}}\\[0.4cm]
\end{longtable}

\hypertarget{publications}{%
\section{Publications}\label{publications}}

\hypertarget{bibliography}{}
\leavevmode\vadjust pre{\hypertarget{ref-5}{}}%
\CSLLeftMargin{1. }
\CSLRightInline{Clark, C. M., Hernandez, A., Mullowney, M. W.,
Fitz-Henley, J., Li, E., Romanowski, S. B., Pronzato, R., Manconi, R.,
Sanchez, L. M., \& Murphy, B. T. (2022). Relationship between bacterial
phylotype and specialized metabolite production in the culturable
microbiome of two freshwater sponges. \emph{ISME Communications.
Citations: 0}.
\href{https://10.1038/s43705-022-00105-8}{10.1038/s43705-022-00105-8}}

\leavevmode\vadjust pre{\hypertarget{ref-4}{}}%
\CSLLeftMargin{2. }
\CSLRightInline{Leao, T. F., Clark, C. M., Bauermeister, A., Elijah, E.
O., Gentry, E. C., Husband, M., Oliveira, M. F., Bandeira, N., Wang, M.,
\& Dorrestein, P. C. (2021). Quick-start infrastructure for untargeted
metabolomics analysis in GNPS. \emph{Nature Metabolism. Citations: 1}.
\href{https://10.1038/s42255-021-00429-0}{10.1038/s42255-021-00429-0}}

\leavevmode\vadjust pre{\hypertarget{ref-7}{}}%
\CSLLeftMargin{3. }
\CSLRightInline{Clark, C. M., Murphy, B. T., \& Sanchez, L. M. (2020). A
call to action: The need for standardization in developing open-source
mass spectrometry-based methods for microbial subspecies discrimination.
\emph{mSystems. Citations: 2}.
\href{https://10.1128/msystems.00813-19}{10.1128/msystems.00813-19}}

\leavevmode\vadjust pre{\hypertarget{ref-6}{}}%
\CSLLeftMargin{4. }
\CSLRightInline{Clark, C. M., Costa, M. S., Conley, E., Li, E., Sanchez,
L. M., \& Murphy, B. T. (2019). Using the open-source MALDI TOF-MS IDBac
pipeline for analysis of microbial protein and specialized metabolite
data. \emph{Journal of Visualized Experiments. Citations: 9}.
\href{https://10.3791/59219}{10.3791/59219}}

\leavevmode\vadjust pre{\hypertarget{ref-1}{}}%
\CSLLeftMargin{5. }
\CSLRightInline{Braesel, J., Clark, C. M., Kunstman, K. J., Green, S.
J., Maienschein-Cline, M., Murphy, B. T., \& Eustáquio, A. S. (2019).
Genome sequence of marine-derived streptomyces sp. Strain F001, a
producer of akashin a and diazaquinomycins. \emph{Microbiology Resource
Announcements. Citations: 0}.
\href{https://10.1128/mra.00165-19}{10.1128/mra.00165-19}}

\leavevmode\vadjust pre{\hypertarget{ref-2}{}}%
\CSLLeftMargin{6. }
\CSLRightInline{Costa, M. S., Clark, C. M., Ómarsdóttir, S., Sanchez, L.
M., \& Murphy, B. T. (2019). Minimizing taxonomic and natural product
redundancy in microbial libraries using MALDI-TOF MS and the
bioinformatics pipeline IDBac. \emph{Journal of Natural Products.
Citations: 11}.
\href{https://10.1021/acs.jnatprod.9b00168}{10.1021/acs.jnatprod.9b00168}}

\leavevmode\vadjust pre{\hypertarget{ref-3}{}}%
\CSLLeftMargin{7. }
\CSLRightInline{Clark, C. M., Costa, M. S., Sanchez, L. M., \& Murphy,
B. T. (2018). Coupling MALDI-TOF mass spectrometry protein and
specialized metabolite analyses to rapidly discriminate bacterial
function. \emph{Proceedings of the National Academy of Sciences.
Citations: 45}.
\href{https://10.1073/pnas.1801247115}{10.1073/pnas.1801247115}}

\end{document}
